Domain Controller/Domain Controller.md

```diff
@@ -44,6 +47,10 @@ Sandra and Andy's company gets acquired by a larger company that has an existing
  * Cockpit must provide a graphical user experience for executing all of the workflows described in this document.
  * Cockpit must be able to export an Ansible playbook at the end of the workflow, prior to executing the changes.
  * Cockpit must be able to import an Ansible playbook for domain creation and execute it. (Expert mode for custom changes made to the Ansible playbook).
+ 
+### Fleet Commander
+ * Fleet Commander must provide an administrative interface as an extension to Cockpit that covers the desktop configuration management workflow
+ * Fleet Commander must provide a fully integrated experience with Cockpit, FreeIPA and possibly Active Directory
```

Comment:
So, this proposal is specifically about the Domain Controller Role. What we need here is a list of requirements that the role must provide in order to support Fleet Commander.

If you want to include Fleet Commander as a role of its own (which I encourage), that would actually be a new, separate proposal.